src/wp-includes/class-wp-scripts.php

```diff
@@ -907,9 +907,19 @@ private function is_delayed_strategy( $strategy ) {
 	 * @return string The best eligible loading strategy.
 	 */
 	private function get_eligible_loading_strategy( $handle ) {
-		$eligible = $this->filter_eligible_strategies( $handle );
+		$intended = (string) $this->get_data( $handle, 'strategy' );
 
-		// Bail early once we know the eligible strategy is blocking.
+		// Bail early if there is no intended strategy.
+		if ( ! $intended ) {
```

Comment:
Strict type checkout would be ideal if always an empty string.

Reply:
Normally, `$this->get_data()` will return `false` here, but it's possible someone has set this data value to be explicitly `false` or  an empty string. In any of those cases we can return early with an empty string.